docs/samples-and-tutorials/index.md

```diff
@@ -32,64 +32,64 @@ online playground, try these [interactive tutorials](http://go.microsoft.com/fwl
 
 **[Unit Testing in .NET Core using dotnet test](../core/testing/unit-testing-with-dotnet-test.md)**
 
-This guide shows how to create an ASP.NET Core web application and the associated unit tests. It will start by creating a simple web service application and then add tests, and continue by creating more tests to guide implementing new features. The completed code is available [in the core-docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/core/getting-started/unit-testing-using-dotnet-test).
+This guide shows how to create an ASP.NET Core web application and the associated unit tests. It will start by creating a simple web service application and then add tests, and continue by creating more tests to guide implementing new features. The completed code is available [in the docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/core/getting-started/unit-testing-using-dotnet-test).
 
 ### Tutorials
 
 **[Writing .NET Core console apps using the CLI tools: A step-by-step guide](../core/tutorials/using-with-xplat-cli.md)**
 
-This guide will show you how to use the .NET Core CLI tooling to build cross-platform console apps.  It will start with the most basic console app and eventually span multiple projects, including testing. You'll add these features step-by-step, building on what you've already seen and built. A finished version of the code you'll build is located [in the core-docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/core/console-apps).
+This guide will show you how to use the .NET Core CLI tooling to build cross-platform console apps.  It will start with the most basic console app and eventually span multiple projects, including testing. You'll add these features step-by-step, building on what you've already seen and built. A finished version of the code you'll build is located [in the docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/core/console-apps).
 
 **[Writing Libraries with Cross Platform Tools](../core/tutorials/libraries.md)**
 
 This sample covers how you can write libraries for .NET using cross-platform CLI tools.  They provide an efficient and low-level experience that works across any supported OS.
-The completed code is available [in the core-docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/framework/libraries/frameworks-library).
+The completed code is available [in the docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/framework/libraries/frameworks-library).
```

Comment:
nit: I just realized that we usually refer to repos as dotnet/docs. Would this look better than just docs?